python/ray/autoscaler/kubernetes/defaults.yaml

```diff
@@ -1,13 +1,13 @@
-# An unique identifier for the head node and workers of this cluster.
-cluster_name: default
+# A unique identifier for the head node and workers of this cluster.
+cluster_name: example-cluster
 
 # The minimum number of workers nodes to launch in addition to the head
 # node. This number should be >= 0.
-min_workers: 0
+min_workers: 2
 
 # The maximum number of workers nodes to launch in addition to the head
 # node. This takes precedence over min_workers.
```

Comment:
Why? This should remain 0 for cost saving.

Reply:
Ah right -- I'll set defaults to 0, but keep example-full to 2 for the sake of
being able to verify autoscaling.

Reply:
I think all our examples should set min_workers to 0 to avoid charging users unnecessary costs.

Reply:
If a test needs min_workers>0, the the test should change it internally.

Reply:
Does that apply to 

> This looks good to me.
> Here is what would be ideal here.
> 
> 1. Try running ray up with all the examples here.
> 2. Attach to the cluster and check if all the logs look sane and the cluster is autoscaling.
>    I think we should have that before merging this PR. We should also add the documentation part (not necessary in this PR).

Yep, I've done the tests.

From user feedback, autoscaling in response to resource loads seems to be failing on K8s, but that's a whole other issue to track. 
(min_workers work and are tested in the unit tests I run locally)

Working on documentation now.

Reply:
> I think all our examples should set min_workers to 0 to avoid charging users unnecessary costs.

Hmm, I'm using `example-full.yaml` in documentation, and I think users like to verify that autoscaling to min workers is actually happening in their clusters. (And we actually need them to test that, given that they're running in different Kubernetes environments e.g. OpenShift.)
The current example operator configs also use nonzero min_workers.

Also, I get the sense that our Kubernetes users tend to have sufficient knowledge to understand the cost implications of spinning up pods in their own Kubernetes clusters.

Any thoughts on min_workers @edoakes or @yiranwang52 ?

Reply:
Setting it to 0, will indicate in docs how to increase min_workers.